python/shipStrawTracking.py

```
@@ -499,8 +510,12 @@ def run_track_pattern_recognition(input_file, geo_file, output_file, method, dy=
                     h["rmse_x"].Fill(rmse_x)
                     h["rmse_y"].Fill(rmse_y)
 
-            except Exception:
-                print("Problem with fitted state.")
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)